### ISSUE TITLE
Tick labels ignored with datetime()

### ISSUE THREAD

**Issue by okomarov** [matlab2tikz/matlab2tikz#383]
When using `datetime()`, TickLabels are set, but the mode remains `auto`:
```matlab
data = rand(10,1);
plotdates = datetime:datetime+numel(data)-1;
plot(plotdates,data)
matlab2tikz('C:\users\ok1011\desktop\m2t_test\plotdates.tex','standalone',true)
```
Result: 
![capture](https://cloud.githubusercontent.com/assets/3731173/4796425/4df02730-5e03-11e4-9ce1-1cf06af9e8b6.PNG)

We could enforce the check for the mode && if ticklabels are empy in (line 4010):

    if strcmp(tickMode, 'auto') && ~m2t.cmdOpts.Results.strict

**Comment by okomarov** [comment]
The axis will have the 'DatetimeDurationPlotAxesListenersManager' hidden property which can be used to determine a manual mode for ticklabels. However, the `double(gca)` conversion drops these hidden and user-added properties, and it's thus difficult to test for such property.

**Comment by okomarov** [comment]
Implemented with #803

**Comment by awesomeindustries** [comment]
With the code of the TS, I get an error in Matlab R2016b and matlab2tikz 1.0.0

```
Error using datetime/horzcat (line 1260)
All inputs must be datetimes or date/time character vectors or date/time
strings.
Error in matlab2tikz>getXYZDataFromLine (line 1493)
        data = [xData(:), yData(:)];
Error in matlab2tikz>drawLine (line 1429)
    [data] = getXYZDataFromLine(m2t, h);
Error in matlab2tikz>handleAllChildren (line 643)
                [m2t, str] = drawLine(m2t, child);
Error in matlab2tikz>drawAxes (line 852)
    [m2t, childrenEnvs] = handleAllChildren(m2t, handle);
Error in matlab2tikz>saveToFile (line 450)
        m2t = drawAxes(m2t, relevantAxesHandle);
Error in matlab2tikz (line 352)
saveToFile(m2t, fid, fileWasOpen);
Error in matlab2tikztest (line 4)
matlab2tikz('./matlab2tikztest.tex','standalone',true)
```
